spec/workers/articles/bust_cache_worker_spec.rb

```diff
@@ -0,0 +1,44 @@
+require "rails_helper"
+
+RSpec.describe Articles::BustCacheWorker, type: :worker do
+  describe "#perform" do
+    let(:article) { FactoryBot.create(:article) }
+    let(:cache_buster) { double }
+    let(:worker) { subject }
+    let(:buster) { "CacheBuster" }
+
+    before do
+      allow(buster).to receive(:constantize).and_return(cache_buster)
+      allow(cache_buster).to receive(:bust_article)
+    end
+
+    context "with cache buster defined" do
+      it "busts cache with defined buster" do
+        new_buster = "NewBuster"
+        allow(new_buster).to receive(:constantize).and_return(cache_buster)
+        allow(cache_buster).to receive(:bust_article)
```

Comment:
not sure why we create a new buster (we can use the one defined above) and call `receive` again on the old buster

Reply:
I like using a new buster object completely bc it validates that we are not using the default and actually reading from the passed in arguments.

Reply:
saw the change, okay!